san/transform.py

```diff
@@ -0,0 +1,32 @@
+import operator
+from functools import reduce
+from san.pandas_utils import convert_to_datetime_idx_df
+
+QUERY_PATH_MAP = {
+    'eth_top_transactions': ['ethTopTransactions']
+}
+
+def path_to_data(idx, query, data):
+    return reduce(operator.getitem, ['query_'+str(idx),]+QUERY_PATH_MAP[query], data)
+    """
+    With this function we jump straight onto the key from the dataframe, that we want and start from there
+    """
+
+def add_transform(function):
+    def decorator(idx, query, data):
+        if query in QUERY_PATH_MAP:
+            print(underscore_to_camelcase(query))
```

Comment:
remove

Reply:
Yes, I just saw, that I should remove a few things, among them are a few imports in some files